### PR TITLE
feature: added an export csv button for import issues

### PR DESCRIPTION
**Pull request by Ryoken**
#### Any background context you want to provide?
see ticket

#### What's this PR do?
adds a button allowing you to download a CSV of imported file issues

#### How should this be manually tested?
import a file with issues

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2960

#### Screenshots (if appropriate)
Single file import issues and downloaded CSV:
Screen Shot 2021-11-10 at 3 39 09 PM
Screen Shot 2021-11-10 at 3 39 24 PM

Single file match issues and downloaded CSV:
Screen Shot 2021-11-10 at 4 04 25 PM
Screen Shot 2021-11-10 at 4 05 43 PM

Multiple files:
Screen Shot 2021-11-10 at 4 07 27 PM

### CODE REVIEW

File: seed/static/seed/js/controllers/data_upload_modal_controller.js [hunk 1]
```diff
@@ -734,6 +734,29 @@ angular.module('BE.seed.controller.data_upload_modal', [])
         });
       };
 
+      $scope.export_issues = function (issues) {
+        let data = ['File Name,Severity,Message'];
```
Comment (macintoshpie) [inline seed/static/seed/js/controllers/data_upload_modal_controller.js:738]: Ooo nice, I like that we can do all of this on the frontend!

**Review by macintoshpie — CHANGES_REQUESTED** [review]
I had a few issues when testing files with schema errors (use a bogus enum somewhere) as well as use-case errors (remove street address for the building). The "warnings" seemed to work great.

Screen Shot 2021-11-15 at 11 24 12 AM
Screen Shot 2021-11-15 at 11 25 35 AM

Screen Shot 2021-11-15 at 11 27 15 AM

**Comment by Ryoken** [comment]
Screen Shot 2021-11-15 at 12 09 30 PM
Screen Shot 2021-11-15 at 12 07 56 PM

**Review by macintoshpie — APPROVED** [review]
💯  👍

**Pull request merged**